2018-September-18/writingSamples/harmata_sample.tex

```diff
@@ -84,56 +84,76 @@
 % end sentence after breathing
 amygdala causes apnea and other abnormalities of breathing.
 % New sentence, start with something catchy (like remarkably!)
-Remarkably, this hypoventilation can persist for minutes even after
-stimulation or seizure in the amygdala has ceased. In previously
-reported patients, apnea was seen during amygdala stimulation, but
-breathing resumed within 15 seconds afterwards and no abnormal
-hypoventilation beyond that time was shown\textsuperscript{1-3}. This
+% use apnea instead of hypoventilation (or vice versa) 
+% (or are the two words describing unique properties?)
+Remarkably, apnea can persist for minutes even after
+% Instead of "In previously", you could say, 
+% "This stands in stark contrast to previous reports"
+stimulation or seizure in the amygdala has ceased. 
+% I don't quite like my suggested sentence either, but maybe it'll be useful.
+This stands in stark contrast to previous reports where normal breathing
+resumed within 15 seconds after amygdala stimulation\textsuperscript{1-3}. This
+% I really like this last sentence.
 novel finding of long-lasting breathing changes may have critical
 implications for understanding postictal mortality.
 
 \emph{Prevalence of the prolonged disrupted breathing phenotype}
```

Comment:
I think the ordering of the paragraphs is good, but in the first paragraph it may be better to set up that there are three reasons why prolonged postictal hypo-ventilation may be under discovered (e.g. the current method of reporting, the specificity of the stimulation/seizure foci, and the connectivity of the temporal lobe).